short_term/database_handler.py

```diff
@@ -3,7 +3,7 @@
 
 from os import environ as ENV
 
-from pymssql import connect
+from pymssql import connect  # pylint: disable=E0611
```

Comment:
What's `E0611`?

Reply:
Disable 'connect doesn't exist' for pymssql library errorwhen it clearly does exist